### DISCUSSION TITLE
Question about the evalution metric for amaon

### DISCUSSION THREAD

**Discussion by yfzhang114** [p-lambda/wilds#114] [Q&A] [answered]
I read your training logs in https://worksheets.codalab.org/bundles/0xe8079c938aeb48afa57b4331e8560f38 and found something confusing. I wonder which metric is the Val 10% Acc/Val Avg Acc and which one is the Test 10% Acc/Test Avg Acc. 

![image](https://user-images.githubusercontent.com/53161950/159099941-b4c9a474-2112-449f-859e-8d364aa4f303.png)

**Comment by ssagawa** [comment, marked as answer]
Below the heading "Validation (OOD)," the log reports the performance on the OOD val split. The test numbers aren't printed in the log, but are reported in the `test_eval.csv` file.
